news/2024/2024-12-22-touhou-project-mania-cup-4-registrations-now-open.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: "Touhou Project Mania Cup 4: Registrations are Now Open!"
+date: 2024-12-22 12:00:00 +0000
+---
+
+![](/wiki/Tournaments/TMC/4th/image/banner@2x.png)
+
+With the celebration of the Touhou Project's 30th birthday, the Touhou Project Mania Cup is returning with its 4th iteration, bringing tons of improvement in competitiveness and player enjoyment!
+
+The [Touhou Project Mania Cup 4th](/community/forums/topics/2015815), or THMC 4 for short, is a team-based osu!mania 4k tournament. The tournament features 100% Touhou-related music and aims to bring top-tier competition to the tournament scene.
+
+We’ve had 80 teams [last year](/wiki/en/Tournaments/TMC/3rd), with **Bloxy cola** reaching the top with an undefeated performance. [This year](/wiki/en/Tournaments/TMC/4th), with more teams joining the competition, which team will rise to the challenge and become the best? 
+
+## What's new
+
+- **A Group Stage will debut this year** featuring a 16-group format. This means that we will have **64 teams** in the main event.
+  - If more than 70 teams play the qualifier, the group stage will use the split double elimination format.
```

Comment:
```suggestion
  - If there are more than 70 teams participating in the Qualifier, the Group Stage will use the split double elimination format.
```

Reply:
(also you might want to explain a bit what is this "split double elimination format" since i don't think i've heard this term elsewhere myself)

Reply:
yeah probably it's better to link that as such

`If there are more than 70 teams participating in the Qualifier, the Group Stage will use the [split double elimination format](http://liquipedia.net/osu/User:RushFTK/Touhou_Project_Mania_Cup/4/Format).`

Reply:
i will link to wiki rules then.